src/test/java/com/cloudbees/jenkins/support/filter/ContentMappingsTest.java

```diff
@@ -165,4 +172,22 @@ public void clear() {
             assertTrue(stopWords.contains(originalVersion));
         });
     }
+
+    /*
+    @Test
+    public void filteredNamesGeneratingBundle() {
+        JenkinsRule.JSONWebResponse jsonWebResponse = rule.postJSON(root.getUrlName() + s, "");
+        IOUtils.copy(jsonWebResponse.getContentAsStream(), Files.newOutputStream(zipFile.toPath()));
+        ZipFile z = new ZipFile(zipFile);
+        zj.
+
+        File zipFile = File.createTempFile("test", "zip");
+        ((File) zipFile).deleteOnExit();
+        FileOutputStream fileOutputStream = new FileOutputStream(tempLogFile);
+
+        ClearCaseChangeLogSet.saveToChangeLog(fileOutputStream, history);
+        fileOutputStream.close();
+
+        comprobar que el zip tiene las entradas bien
+    }*/
```

Comment:
Dead code

Reply:
Why are you bringing out all my playground?

Reply:
I leave the same stuff in my last PR, I know at which point it's easy to forget about those stuff :D